pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -19,11 +19,28 @@ package v1alpha1
 import (
 	"context"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
 	"knative.dev/pkg/apis"
 )
 
 // Validate EventListener.
 func (t *EventListener) Validate(ctx context.Context) *apis.FieldError {
+	clientset := ctx.Value("clientSet").(dynamic.Interface)
+
+	triggerBindings := SchemeGroupVersion.WithResource("triggerBindings")
```

Comment:
Should this be upper camel case?

Reply:
Done

Reply:
I was referring to the resource parameter to the `WIthResource()` method. Seems like the convention is to use lowercase. Can you also add a validation test file for this?

Reply:
@vtereso 
Right, it's lowercase, it's very old version, has already overwrite by `commit --amend` for long time.
Do you means we need add test file for the validation?

Reply:
I haven't realized on my initial review that it did not contain tests for the validation. @dibyom do think we can do without them?